SafeguardSessions/README.md

```diff
@@ -148,7 +156,32 @@ When writing a release note, use the following template:
 
 If a section does not have related content, it can be left out.
 
-Modifications to the Report Template must be listed in the appropriate sections, since they are part of the release.
+Modifications to the Report Template are a part of the release, and must be listed in the appropriate sections. To solve the issue, make sure your modifications are listed in the appropriate sections.
+
+## Refreshing your preloaded data
+
+Using a .pbix file, you can refresh your previously downloaded data without having to create a new data source.
+
+If you are a new user and haven't used the custom connector yet, we strongly suggest that you follow the steps of [Installation and usage].
+
+If you want to upgrade your custom connector, see [Upgrading the Power BI Connector].
+
+**To refresh your preloaded data**
+1. Open your previously saved .pbix file in Power BI Desktop.
+2. In Power BI Desktop, click **Home > Refresh**.
+
+If an error occurs after refreshing your data, discard your changes to keep your data intact.
+
+## Upgrading the Power BI Connector
+
+If you are a new user and haven't used the custom connector yet, we strongly suggest that you follow the steps of [Installation and usage].
+
+You can safely upgrade to a new version of the Power BI Connector if the major version number of the new Power BI Connector does not differ from the one you are currently using. If the major version is different, we strongly suggest that you view the Breaking changes section for the new Power BI Connector on the [Releases] page.
```

Comment:
I know in the [About the project] section we wrote that the Info table holds the connector version number, but if the user did not read that part, just got here straight, we should also give here a clue, where the he/she can check the current connector version.

Reply:
Done